crates/oxc_linter/src/lib.rs

```diff
@@ -8,6 +8,8 @@ mod ast_util;
 mod config;
 mod context;
 mod disable_directives;
+mod env;
+mod environments;
```

Comment:
These two mean the same thing, I'll rename these to something else once this is merged.

Reply:
changed to `mod javascript_globals;` based on the previous suggestion 👀.